fixtures/dummy/app/views/comments/comment.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Views
+	module Comments
+		class Comment < Phlex::Component
+			def initialize(name:, body:)
+				@name = name
+				@body = body
+			end
+
+			def template(&block)
```

Comment:
I tried using just `&` here, but the Rubocop is configured for Ruby 2.7 version and it throws errors.

We can have another issue/PR to support Ruby 3.1 version for Rubocop inside the project, WDYT?

Reply:
We use 2.7 style in the library and test files because we need to support Ruby 2.7. Documentation is built in 3.1.2 so we use the new style for that.